entries/f/fanatics.com.json

```
@@ -0,0 +1,12 @@
+{
+  "Fanatics": {
+    "domain": "fanatics.com",
+    "tfa": [
+      "sms",
+      "email"
+    ],
+    "categories": [
+      "retail"
+    ]
+  }
```

Looks like fanatics.com is U.S. only.
```suggestion
  },
  "regions": [
    "us"
  ]
```